backend/backend.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+
+apt-get update
+
+
+#need in swtpm 
+apt -y install autoconf-archive libcmocka0 libcmocka-dev procps iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev autoconf doxygen libjson-c-dev libini-config-dev libcurl4-openssl-dev libltdl-dev expect python3-yaml libglib2.0-dev dbus  libtasn1-6-dev libjson-glib-dev gawk socat libseccomp-dev uuid-dev udev libusb-1.0-0-dev libgnutls28-dev gnutls-bin
+
+
+#need in acs
+apt-get install -y libjson-c4  libjson-c-dev mariadb-server libmysqlclient-dev git make gcc wget 
+rm -rf /var/lib/apt/lists/*
```

Comment:
`gcc ` is installed in line 8 and line 12. please check the same problem

Reply:
Appearing in two places is to quickly find all the dependencies for the subsequent replacement of the component location